src/inference/src/system_conf.cpp

```diff
@@ -88,13 +93,17 @@ bool with_cpu_x86_avx512_core_vnni() {
 }
 
 bool with_cpu_x86_bfloat16() {
-    return get_cpu_info().has(Xbyak::util::Cpu::tAVX512_BF16);
+    return get_cpu_info().has(Xbyak::util::Cpu::tAVX512_BF16) || with_cpu_x86_avx2_vnni_2();
```

Comment:
I don't agree with this change. avx2_vnni_2 doesn't add any bf16 compute, there are only additional upconvert instructions (which usage is very limited in the product). So I would keep compute `meaning` for both `with_cpu_x86_bfloat16` and `with_cpu_x86_float16` in public API.

Reply:
reverted this changes